spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -1,43 +1,43 @@
 require "rails_helper"
 
 describe "Posters API" do
-  it "sends a list of posters" do
-    Poster.create(
-      name: "REGRET",
-      description: "Hard work rarely pays off.",
-      price: 89.00,
+  before(:each) do
```

Comment:
Adding the before:each block really cleans up the entire test file. This is a really nice refactor.